premake4.lua

```diff
@@ -0,0 +1,30 @@
+solution "litehtml"
+    configurations { "Release", "Debug" }
+
+    targetname "litehtml"
+    language "C++"
+    kind "StaticLib"
+
+    files
+    {
+        "src/**.cpp", "src/**.h"
+    }
+
+    configuration "Debug"
+        defines     { "_DEBUG" }
+        flags       { "Symbols" }
+
+    configuration "Release"
+        defines     { "NDEBUG" }
+        flags       { "OptimizeSize" }
+
+    configuration "windows"
+        defines     { "WIN32" }
+
+    project "ShaderShaker"
```

Comment:
except for the project "ShaderShaker" part i think it is... could you elaborate?